packages/plugins/rubygem-smart_proxy_pulp/rubygem-smart_proxy_pulp.spec

```diff
@@ -60,6 +61,10 @@ mkdir -p  %{buildroot}%{_sysconfdir}/foreman-proxy/settings.d/
 cp -pa .%{gem_instdir}/settings.d/pulp.yml.example %{buildroot}%{foreman_proxy_settingsd_dir}/pulp.yml
 cp -pa .%{gem_instdir}/settings.d/pulpnode.yml.example %{buildroot}%{foreman_proxy_settingsd_dir}/pulpnode.yml
 
+#copy cron scripts to be scheduled
+install -d -m0755 %{buildroot}%{_sysconfdir}/cron.d
+install -m 644 %{SOURCE1} %{buildroot}%{_sysconfdir}/cron.d/%{name}
```

Comment:
Should this be `install -D` and drop the line above it?

Reply:
probably. copy&paste from katello.spec ;)